tools/check_format.py

```diff
@@ -766,11 +766,11 @@ def checkFormat(file_path):
   if isBuildFile(file_path) or isSkylarkFile(file_path) or isWorkspaceFile(file_path):
     if try_to_fix:
       error_messages += fixBuildPath(file_path)
-    error_messages += checkBuildPath(file_path)
+    #error_messages += checkBuildPath(file_path)
```

Comment:
Revert? (I assume you are debugging the same issue I am dealing with)

Reply:
Done (yes)